src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -42,9 +43,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException {
     try {
       String url = request.getParameter(URL_PARAMETER);
-      // TODO: add numComments parameter. Comment count will be hard-coded until then.
+      String numCommentsInput = request.getParameter(NUM_COMMENTS_PARAMETER);
+      // If nothing is inputted, length of list is default, otherwise it's what was inputted.
+      long numComments =
+          (numCommentsInput == null || numCommentsInput.trim().equals(""))
+              ? DEFAULT_COMMENTS
+              : Long.valueOf(numCommentsInput);
```

Comment:
Maybe for simplicity we could chance the input box in the HTML from a user typed input to a slider

Reply:
I personally feel like it's easier just to have users input the number of comments explicitly, since from what I've seen a slider doesn't inherently show the exact number.

Reply:
hmmm I think it does? https://www.w3schools.com/howto/howto_js_rangeslider.asp

Reply:
It doesn't do it by default but it's certainly possible, and that article explains how to do it